hadoop-tools/hadoop-benchmark/pom.xml

```diff
@@ -22,11 +22,11 @@
   <parent>
     <groupId>org.apache.hadoop</groupId>
     <artifactId>hadoop-project</artifactId>
-    <version>3.4.0-SNAPSHOT</version>
+    <version>3.3.9-SNAPSHOT</version>
     <relativePath>../../hadoop-project/pom.xml</relativePath>
   </parent>
   <artifactId>hadoop-benchmark</artifactId>
-  <version>3.4.0-SNAPSHOT</version>
+  <version>3.3.9-SNAPSHOT</version>
```

Comment:
Just a note: this was introduced in 5c348c41ab8ddb81146355570856e61e8d129a1e which is a backport of #3904. That PR is adding a new module, `hadoop-benchmark`, and the version numbers weren't updated when it was backported, so it still points to `3.4.0` (the version used by `trunk`).

I would have expected `branch-3.3` to be on `3.3.5`, but it appears from the discussion in #4482 that `3.3.9` is expected.

Reply:
It should track the current branch, so 3.3.9 is correct. This was fixed by HADOOP-18322 in 7eb1c908a0 .